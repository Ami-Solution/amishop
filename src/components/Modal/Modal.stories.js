/* eslint-disable */
import React from 'react'
import styled, { css } from 'styled-components'
import { storiesOf, addDecorator } from '@storybook/react'
import { withInfo } from '@storybook/addon-info'
import Modal from './index'

storiesOf('03 - Components', module)
  .add(
    'Modal',
    withInfo()(() => (
      <Modal closeFunc={()=>{}}>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nisi, sunt ea aliquam in aliquid voluptates amet asperiores corrupti, aspernatur veritatis exercitationem, quas dignissimos molestias dolorem. Beatae itaque tempora doloremque, accusamus.
      </Modal>
    ))
  )
