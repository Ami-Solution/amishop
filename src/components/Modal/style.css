.title-stl {
  color: #000000;
}

.content-stl {
  margin-top: 1vh;
  margin-bottom: 1vh;
}

.modal-stl {
  display: flex;
  flex-flow: column;
  justify-content: space-around;
  text-align: center;
  height: 50vh;
  border-radius: 1em;
  box-shadow: 0px 0px 10px;
  background: white;
  width: 90%;
  margin: auto;
  font-family: helvetica;
  font-weight: 300;
  color: dimgrey;
  padding-top: 2vh;
  padding-bottom: 2vh;
}

.overlay-stl {
  height: 100vh;
  position: fixed;
  width: 100%;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background-color: rgba(0,0,0,0.5);
  z-index: 2;
  cursor: pointer;
  display: flex;
  flex-flow: column;
  justify-content: center;
}
