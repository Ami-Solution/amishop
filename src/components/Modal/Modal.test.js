/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Modal from './index';

configure({ adapter: new Adapter() });

describe('Component Modal', () => {
  const component = renderer
    .create(<Modal><p>Modal</p><p>Home</p></Modal>);
  const tree = component.toJSON();

  it('should be render without crash', () => {
    expect(tree).toMatchSnapshot();
  });
});
