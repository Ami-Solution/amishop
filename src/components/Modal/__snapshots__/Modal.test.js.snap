// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Modal should be render without crash 1`] = `
<div
  className="sc-gzVnrw gVHaxn"
>
  <div
    className="sc-ifAKCX KRuoF"
  >
    <div
      className="sc-EHOje lfAdil"
    >
      <p>
        Modal
      </p>
      <p>
        Home
      </p>
    </div>
  </div>
</div>
`;
